config/migrations.php

```diff
@@ -85,6 +85,6 @@
         | The length for the version column in the migrations table.
         |
         */
-        'version_column_length' => 14
+        'version_column_length' => 1024
```

Comment:
@seifane-wise Do you remember why you set 1024 here and not 191? See updated default her: https://github.com/doctrine/migrations/blob/3.5.x/UPGRADE.md

Reply:
I know 1024 is used in the docs https://github.com/doctrine/migrations/blob/3.5.x/docs/en/reference/configuration.rst but its actually 191 by default in thier code